### PR TITLE
reword the 'round' definition

### PR DESCRIPTION
**Pull request by laser**
## Why is this PR needed?

The previous wording was confusing. It was not clear (to me) that the definition of "round" applied to a period of time in which leader election occurred but no leader was found.

## What's in this PR?

I've added more words to the "round" definition in hopes of reducing ambiguity.

### CODE REVIEW

**Comment by dignifiedquire** [comment]
@laser want to take a stab at improving this further?

**Comment by sternhenri** [comment]
May take a stab if you're ok with it @laser @dignifiedquire given conflicting requests in https://github.com/filecoin-project/specs/issues/357 (cc @whyrusleeping @pooja)

**Comment by laser** [comment]
@sternhenri - Please do! I would very much appreciate that.

**Comment by sternhenri** [comment]
- Made round and height synonymous.
    - Did not remove round from spec entirely as it is easier to use in some sentences (e.g. miner runs another round vs. miner tries again at the next height --> means same thing, first less awkward)
- removed mentioned of epochs (in consensus context)
- likewise removed mentions of null blocks (nonsensical as they stood half-unused) in favor of unified 'losing tickets' solution.

Ready to merge from my end.

File: definitions.md [hunk 1]
```diff
@@ -109,7 +105,11 @@ The Generation Attack Threshold is equal to the Polling Time + some Grace Period
 
 #### Height
 
-`Height` refers to the number of `TipSets` that have passed between this `TipSet` and the genesis block (which starts at block height 0). If a `TipSet` contains multiple blocks, each block in the TipSet will have the same `height`.
+`Height` and `round` are synonymous and used interchangeably in this spec.
+
+`Height` refers to the number of tickets generated between this `TipSet` and the genesis block (height 0), counting only a single ticket per TipSet.
```
Comment (dignifiedquire) [inline definitions.md]: the last part of this sentence is still confusing at best, wrong at worst.

File: mining.md [hunk 2]
```diff
@@ -335,6 +334,7 @@ Heaviest tipset at H-1 is {B0}
 - New Round with M mining atop B4
 
 Anytime a miner receives new blocks, it should evaluate which is the heaviest TipSet it knows about and mine atop it.
+>>>>>>> ca3f93b3497b870b513332d3cfee381b7590e993
```
Comment (dignifiedquire) [inline mining.md]: merge conflict detected

**Pull request merged**